morebits.js

```diff
@@ -800,6 +785,41 @@ Morebits.quickForm.element.prototype.compute = function QuickFormElementCompute(
 				removeButton.morebutton = data.morebutton;
 			}
 			break;
+		case '_dyninput_cell': // Private, similar to normal input
```

Comment:
for my understanding, why did this case move down?

Reply:
It didn't move down, just renamed.

Reply:
`_dyninput_element` was an internal type used by `dyninput`. So in the code `dyninput` was followed by `_dyninput_element`. Now `_dyninput_cell` is an internal type used by `_dyninput_row`, which is in turn used by `dyninput`. So in the code `dyninput` is followed by `_dyninput_row`, which is followed by `_dyninput_cell`.